frontend/javascripts/oxalis/view/action-bar/toolbar_view.js

```diff
@@ -156,6 +170,49 @@ function OverwriteModeSwitch({ isControlPressed, isShiftPressed, visible }) {
   );
 }
 
+function AdditionalSkeletonModesButtons() {
+  const dispatch = useDispatch();
+  const isMergerModeEnabled = useSelector(
+    state => state.temporaryConfiguration.isMergerModeEnabled,
+  );
+  const isNewNodeNewTreeModeOn = useSelector(state => state.userConfiguration.newNodeNewTree);
+  const toggleNewNodeNewTreeMode = () =>
+    dispatch(updateUserSettingAction("newNodeNewTree", !isNewNodeNewTreeModeOn));
+  const toggleMergerMode = () => dispatch(setMergerModeEnabledAction(!isMergerModeEnabled));
+  const activeButtonStyle = { ...narrowButtonStyle, borderColor: "#1890ff" };
```

Comment:
Did you check the border color in both themes? Ideally, this would use an antd variable, such as `--ant-box-shadow-base`.

Reply:
good catch :)